packages/realm-server/tests/realm-server-test.ts

```diff
@@ -2097,12 +2089,107 @@ module('Realm Server', function (hooks) {
           .set('Accept', 'application/vnd.api+json')
           .set(
             'Authorization',
-            `Bearer ${createJWT(testRealm, 'john', ['read'])}`,
+            `Bearer ${createJWT(testRealm, 'john', [
+              'read',
+              'write',
+              'realm-owner',
+            ])}`,
           );
 
         assert.strictEqual(response.status, 200, 'HTTP 200 status');
+        let json = response.body;
+        assert.deepEqual(
+          json,
+          {
+            data: {
+              id: testRealmHref,
+              type: 'realm-info',
+              attributes: { ...testRealmInfo, visibility: 'private' },
+            },
+          },
+          '/_info response is correct',
+        );
       });
     });
+
+    module(
+      'shared realm because there is `users` permission',
+      function (hooks) {
+        setupPermissionedRealm(hooks, {
+          users: ['read', 'write'],
+          john: ['read', 'write', 'realm-owner'],
+          'test-worker': ['read', 'write', 'realm-owner'],
+        });
+
+        test('200 with permission', async function (assert) {
+          let response = await request
+            .get(`/_info`)
+            .set('Accept', 'application/vnd.api+json')
+            .set(
+              'Authorization',
+              `Bearer ${createJWT(testRealm, 'john', [
+                'read',
+                'write',
+                'realm-owner',
+              ])}`,
+            );
+
+          assert.strictEqual(response.status, 200, 'HTTP 200 status');
+          let json = response.body;
+          assert.deepEqual(
+            json,
+            {
+              data: {
+                id: testRealmHref,
+                type: 'realm-info',
+                attributes: { ...testRealmInfo, visibility: 'shared' },
+              },
+            },
+            '/_info response is correct',
+          );
+        });
+      },
+    );
+
+    module(
+      'shared realm because there are non-realm owner users',
+      function (hooks) {
+        setupPermissionedRealm(hooks, {
+          bob: ['read'],
+          jane: ['read'],
+          john: ['read', 'write', 'realm-owner'],
+          'test-worker': ['read', 'write', 'realm-owner'],
```

Comment:
if this is a realm bot make sure to prefix with `realm/`

Reply:
I removed the realm bot because I don't need that in test. The test uses node-test_realm user for realm bot.